docs/install.html

```diff
@@ -119,12 +120,111 @@ <h2><a name="installation"></a>Installation</h2>
 		is NOT recommended to set the directory to full write, 0777.
 	</p>
 
+	<h3>Configuration</h3>
+
+	<p>
+		WebSVN always requires a special configuration file, for which a template is provided as
+		<code>include/distconfig.php</code>. Only the things mentioned in that template are valid
+		configs at all and some of those are even necessary, e.g. WebSVN obviously needs to know
+		which SVN-repos it should publish. Making the necessary config file available to WebSVN can
+		be done using two ways: Either simply by copying the template file to the hard-coded path
+		<code>include/config.php</code> AND/OR by setting the environment variable
+		<code>WEBSVN_PATH_CONF</code> in a way so that PHP can read it's value. That value is
+		expected to be the path to the config file to use and things like symlinks work as well of
+		course, as long as those are handled transparently by the file system.
+	</p>
+
+	<p>
+		Using the environment variable has two main benefits: One doesn't neet to store additional
+		files in the deployment of WebSVN and the path given by <code>WEBSVN_PATH_CONF</code> is
+		used ADDITIONALLY to <code>include/config.php</code>, if that file is present at all. If
+		not, the file given by the environment variable is used exclusively. This approach allows
+		to override specially chosen configs of some main configuration file using another one
+		based on things like the current location of a request to WebSVN. A webserver like Apache
+		httpd could be configured using <code>Location</code>-blocks to provide different paths to
+		config files using the environment variable like in the following example:
+	</p>
+
+	<pre>
+&lt;Location "/websvn/[...]"&gt;
+	SetEnv WEBSVN_PATH_CONF "/home/websvn/config/[...]/config.php"
+&lt;/Location&gt;</pre>
+
+	<p>
+		Considering both the hard-coded config file path and the more dynamic environment variable
+		e.g. allows setups in which one installation of WebSVN hosts various different SVN-repos
+		configured using <code>$config->parentPath(...)</code>. That statement would simply be added
+		the those files only provided by the environment variable and depending on the location of
+		some request, while all other common things like available templates, syntax highlighting
+		etc. would be set in <code>include/config.php</code>. That allows structuring SVN-repos in
+		subdirs like supported by <code>svnserve</code>, which is otherwise not supported by WebSVN,
+		as it checks if each directory is an SVN-repo already or simply ignores it. The following is
+		a more complete example:
```

Comment:
There is comparative of complete, it is either complete or incomplete

Reply:
Is "comprehensive" a better term? The example doesn't contain everything, only more htan the former one. Or better use "verbose"?

Reply:
Comprehensive is good.